src/client/java/minicraft/level/tile/Tiles.java

```diff
@@ -27,7 +27,27 @@ public static void initTileList() {
 
 		tiles.put((short)0, new GrassTile("Grass"));
 		tiles.put((short)1, new DirtTile("Dirt"));
-		tiles.put((short)2, new FlowerTile("Flower"));
+		tiles.put((short)2, new FlowerTile("Oxeye Daisy", "oxeye_daisy"));
+		// These are out of order because of additions of new flowers
+		tiles.put((short)64, new FlowerTile("Rose", "rose"));
+		tiles.put((short)65, new FlowerTile("Sunflower", "sunflower"));
+		tiles.put((short)66, new FlowerTile("Allium", "allium"));
+		tiles.put((short)67, new FlowerTile("Blue Orchid", "blue_orchid"));
+		tiles.put((short)68, new FlowerTile("Cornflower", "cornflower"));
+		tiles.put((short)69, new FlowerTile("Dandelion", "dandelion"));
+		tiles.put((short)70, new FlowerTile("Hydrangea", "hydrangea"));
+		tiles.put((short)71, new FlowerTile("Iris", "iris"));
+		tiles.put((short)72, new FlowerTile("Orange Tulip", "orange_tulip"));
+		tiles.put((short)73, new FlowerTile("Pink Tulip", "pink_tulip"));
+		tiles.put((short)74, new FlowerTile("Red Tulip", "red_tulip"));
+		tiles.put((short)75, new FlowerTile("White Tulip", "white_tulip"));
+		tiles.put((short)76, new FlowerTile("Peony", "peony"));
+		tiles.put((short)77, new FlowerTile("Periwinkle", "periwinkle"));
+		tiles.put((short)78, new FlowerTile("Pink Lily", "pink_lily"));
+		tiles.put((short)79, new FlowerTile("White Lily", "white_lily"));
+		tiles.put((short)80, new FlowerTile("Poppy", "poppy"));
+		tiles.put((short)81, new FlowerTile("Violet", "violet"));
```

Comment:
As I have said before. Don't create new tiles -- make these variants of the same tile.

I have never given persmission to create new tiles. Especially for something as small as this.

Reply:
You still haven't fixed this. You can't resolve it before fixing it. I havew already said you don't have the permission to add any new tiles.

Reply:
It would not make any sense if only rose tiles drop rose items and flower tiles of other or all flower models/varieties drop the same "flower" items because they are made into dyes. I would rather make all the flower tiles the same and make the game more monotonous instead. Is this what you want?

Reply:
Change the texture of the item to make it look like it could be any flower. You're making a problem out of nothing.

Reply:
Then the placement of the item would not be as same as the original tiles and the recipes of dyes from flowers would not work anymore.

Reply:
Why can't you just make things simple and let it drop a random flower when you break it? The game doesn't need any more shit cluttering up the screen.

Reply:
Then flowers would not be placeable and used as tile decoration.

Reply:
I would suggest having a flower tile that drops a random flower and allowing the placement of said flower with the variants, as a compromise.

Reply:
> Then flowers would not be placeable and used as tile decoration.

Yes, it can. Just make them TileItems. One item, one tile. Different flower variants for decoration.

Reply:
> I would suggest having a flower tile that drops a random flower and allowing the placement of said flower with the variants, as a compromise.

This is what I proposed.

Reply:
Then there will be 2 tiles. One for flower, one for generation, kind of unnecessary.

Reply:
Just one kind of flower tile throughout the whole game would be enough. No need for "random" variation on the same tile.

Reply:
> Then there will be 2 tiles. One for flower, one for random flower, kind of unnecessary.

No.

Reply:
You have messed up here. I will combine the items into just variations of flower item. And as a result, there will be only 1 item and 1 tile for flowers, but they will be handled as variations by using item and tile data.